
<!--
**hoelzer/hoelzer** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- üî≠ I‚Äôm currently working on ...
- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->

## Hi there! üëã
![](https://komarev.com/ghpvc/?username=hoelzer1000&color=blue)

I am a researcher and team lead at the [Genome Competence Center](https://www.rki.de/EN/Content/Institute/DepartmentsUnits/MF/MF1/mf1_node.html) of the [Robert Koch Institute](https://www.rki.de/EN/Home/homepage_node.html) in Berlin, Germany. We study infectious diseases in a One Health context by applying real-time nanopore sequencing, pangenomics approaches, analyzing environmental samples such as wastewater, and studying microbial evolution using bioinformatics for genomic surveillance.

I studied Bioinformatics at the University of Jena, Germany and achieved my PhD by specializing in [RNA Bioinformatics and High-Throughput Analysis](https://www.rna.uni-jena.de/). Following this, I moved to the European Bioinformatics Institute (EMBL-EBI), Hinxton (UK) to work in [Microbiome Informatics](https://www.ebi.ac.uk/about/teams/microbiome-informatics/). Since 2020, I have continued my academic journey at the RKI in Berlin, the national institute for public health in Germany, conducting research with my team and (inter)national colleagues in the following areas (further details can be found on [my personal website](https://hoelzer.github.io/)):

### Research interests üî≠

* Microbial bioinformatics
* Genomic surveillance
* Pathogen detection and characterization
* Nanopore real-time sequencing
* Transcriptomics and gene expression
* Metagenomics and metatranscriptomics
* Cloud computing, automation, and reproducibility with Nextflow and containers

### Skills  :heavy_check_mark:
![My Skills](https://skillicons.dev/icons?i=linux,powershell,bash,r,ruby,html,markdown,git,github,docker,latex)[![nextflow](https://avatars.githubusercontent.com/u/6698688?s=48&v=4)](https://nextflow.io/)

![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=hoelzer&layout=compact&theme=dark)

### Bioinformatics tools :floppy_disk:

My team and I develop bioinformatics tools and pipelines for microbial genomics, nanopore sequencing, metagenomics, and transcriptomics. I favor Nextflow pipelines with containerized dependencies for reproducibility. You can find my code on [my personal GitHub](https://github.com/hoelzer), at [github.com/hoelzer-lab](github.com/hoelzer-lab) (from my time leading a team at the University of Jena), and [github.com/rki-mf1](github.com/rki-mf1) (from the RKI Genome Competence Center). We also contribute to other repositories. Here are some examples:

* [`RIBAP`](https://github.com/hoelzer-lab/ribap): A comprehensive bacterial core gene-set annotation pipeline based on Roary and pairwise ILPs. [[Paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03312-9)
* [`POCP-nf`](https://github.com/hoelzer/pocp): Calculation of the Percentage of Conserved Proteins following Qin, Xie _et al_. [[Paper]](https://academic.oup.com/bioinformatics/article/40/4/btae175/7638801?login=false)
* [`RNAflow`](https://github.com/hoelzer-lab/rnaflow): A simple yet effective bulk RNA-Seq differential gene expression pipeline using Nextflow. [[Paper](https://www.mdpi.com/2073-4425/11/12/1487)]
* [`VIRify`](https://github.com/EBI-Metagenomics/emg-viral-pipeline): VIRify: detection of phages and eukaryotic viruses from metagenomic and metatranscriptomic assemblies. [[Paper]](https://doi.org/10.1101/2022.08.22.504484)
* [`WhatThePhage`](https://github.com/replikation/What_the_Phage): WtP: Phage identification via nextflow and docker or singularity. [[Paper]](https://doi.org/10.1093/gigascience/giac110)
* [`poreCoV`](https://github.com/replikation/poreCov): SARS-CoV-2 genome reconstruction workflow for nanopore sequence data. [[Paper]](https://www.frontiersin.org/articles/10.3389/fgene.2021.711437/full)
* [`CoVpipe2`](https://github.com/rki-mf1/CoVpipe2): SARS-CoV-2 genome reconstruction for Illumina data in Nextflow. [[Paper]](https://doi.org/10.12688/f1000research.136683.1)
* [`CLEAN`](https://github.com/rki-mf1/clean): A nextflow pipeline for decontamination of short reads, long reads and contigs. [[Paper]](https://doi.org/10.1101/2023.08.05.552089)

![stats](https://github-readme-stats.vercel.app/api?username=hoelzer&show_icons=true&theme=dark)

### Selected publications :scroll:

* Eva A√ümann, Shelesh Agrawal, Laura Orschler, Sindy B√∂ttcher, Susanne Lackner, **Martin H√∂lzer** (2024). "Impact of reference design on estimating SARS-CoV-2 lineage abundances from wastewater sequencing data". _GigaScience_.[doi.org/10.1093/gigascience/giae051](https://doi.org/10.1093/gigascience/giae051)
* Kevin Lamkiewicz, Lisa-Marie Barf, Konrad Sachse, **Martin H√∂lzer** (2024). "RIBAP: a comprehensive bacterial core genome annotation pipeline for pangenome calculation beyond the species level". _Genome Biology_, 25 (1), 170.[doi.org/10.1186/s13059-024-03312-9](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03312-9)
* **Martin H√∂lzer** (2024). "POCP-nf: an automatic Nextflow pipeline for calculating the percentage of conserved proteins in bacterial taxonomy". _Bioinformatics_, 40 (4), btae175.[doi.org/10.1093/bioinformatics/btae175](https://doi.org/10.1093/bioinformatics/btae175)
* Guillermo Rangel-Pineros, Alexandre Almeida, Martin Beracochea, Ekaterina Sakharova, Manja Marz, Alejandro Reyes Mu√±oz, **Martin H√∂lzer**, Robert D Finn (2023). "VIRify: an integrated detection, annotation and taxonomic classification pipeline using virus-specific protein profile hidden Markov models." _PLOS Computational Biology_, 19 (8), e1011422.[doi.org/10.1371/journal.pcbi.1011422](https://doi.org/10.1371/journal.pcbi.1011422)
* Sara Saheb Kashaf, Diana M Proctor, Clay Deming, Paul Saary, **Martin H√∂lzer**, Monica E Taylor, Heidi H Kong, Julia A Segre, Alexandre Almeida, Robert D Finn (2021). "Integrating cultivation and metagenomics for a multi-kingdom view of skin microbiome diversity and functions." _Nature Microbiology_, 7 (1), 169-179.[nature.com/articles/s41564-021-01011-w](https://www.nature.com/articles/s41564-021-01011-w)
* Christian Brandt, Sebastian Krautwurst, Riccardo Spott, Mara Lohde, Mateusz Jundzill, Mike Marquet, **Martin H√∂lzer** (2021). "poreCov - an easy to use, fast, and robust workflow for SARS-CoV-2 genome reconstruction via nanopore sequencing." _Frontiers in Genetics_, 12, 711437.[doi.org/10.3389/fgene.2021.711437](https://doi.org/10.3389/fgene.2021.711437)
* Will A Overholt, **Martin H√∂lzer**, Patricia Geesink, Celia Diezel, Manja Marz, Kirsten K√ºsel (2020). "Inclusion of Oxford Nanopore long reads improves all microbial and viral metagenome‚Äêassembled genomes from a complex aquifer system." _Environmental microbiology_, 22(9), 4000-4013.[doi.org/10.1111/1462-2920.15186](https://doi.org/10.1111/1462-2920.15186)
* **Martin H√∂lzer**, Manja Marz (2019). "_De novo_ transcriptome assembly: A comprehensive cross-species comparison of short-read RNA-Seq assemblers". _GigaScience_, 8 (5), giz039.[doi.org/10.1093/gigascience/giz039](https://doi.org/10.1093/gigascience/giz039)

... [Full list of publications](https://scholar.google.de/citations?user=DMZ7Hc8AAAAJ&hl=en)

### How to reach me üì´

[![linkedin](https://skillicons.dev/icons?i=linkedin)](https://www.linkedin.com/in/martin-h%C3%B6lzer-39304018a/)
[![researchgate](https://upload.wikimedia.org/wikipedia/commons/thumb/5/5e/ResearchGate_icon_SVG.svg/50px-ResearchGate_icon_SVG.svg.png)](https://www.researchgate.net/profile/Martin-Hoelzer)

Further details can be found on [my personal website](https://hoelzer.github.io/).

### Acknowledgement üå±

My GitHub landing page was heavily inspired by [github.com/jonas-fuchs](https://github.com/jonas-fuchs) - also check out Jonas' page and the great bioinformatics tools he's developing for viruses! 






